src/connector/src/source/kafka/enumerator/client.rs

```diff
@@ -115,6 +115,69 @@ impl SplitEnumerator for KafkaSplitEnumerator {
 }
 
 impl KafkaSplitEnumerator {
+    pub async fn list_splits_batch(
+        &mut self,
+        expect_start_timestamp_millis: Option<i64>,
+        expect_stop_timestamp_millis: Option<i64>,
+    ) -> anyhow::Result<Vec<KafkaSplit>> {
+        let topic_partitions = self.fetch_topic_partition().await.map_err(|e| {
+            anyhow!(format!(
+                "failed to fetch metadata from kafka ({}), error: {}",
+                self.broker_address, e
+            ))
+        })?;
+        let mut expect_start_offset = if let Some(ts) = expect_start_timestamp_millis {
+            Some(
+                self.fetch_offset_for_time(topic_partitions.as_ref(), ts)
```

Comment:
This is waiting for implementation?

Reply:
we will only support kafka for now. Other connectors are pending